### DISCUSSION TITLE
umfassende Zusammenstellung der Listen von Datenpunkt-Adressen rechts verlinken?!

### DISCUSSION THREAD

**Discussion by philippoo66** [openv/openv#595] [General]
Moin @speters !

Basierend auf der Arbeit von Markus @sarnau Fritze habe ich eine [umfassende Zusammenstellung der Listen von Optolink Datenpunkt-Adressen](https://github.com/philippoo66/ViessData21/blob/master/DP_Listen_2.zip) erstellt. Offensichtlich kommen Viele nicht darauf, seine Tools zu benutzen oder haben den Weg zu den xml Files nicht gefunden oder sonstwas und fragen/suchen immer noch...

Wäre es nicht sinnvoll, den Link auf die Sammlung in openv rechts in der Linkleiste aufzunehmen?

![grafik](https://user-images.githubusercontent.com/122479122/235305361-d5c6a201-f1b1-4f1f-9436-71e77d0514ee.png)

und vielleicht auch noch den auf [Viessdata21](https://github.com/philippoo66/ViessData21) unter Windows Software...

beste Grüsse!
Phil

**Comment by speters** [comment]
Vielen Dank für den Hinweis!
Ist nun in der Sidebar vom Wiki aufgenommen.

Wiki-Einträge stehen allen Github-Nutzern offen und können gerne ergänzt werden.

**Comment by philippoo66** [comment]
top, danke! 
dass die jeder ergänzen kann, wusste ich nicht, und ich wüsste immer noch nicht, wie das geht. aber wahrscheinlich hilft google oder chatgpt da weiter... ;-)
